api/desecapi/templates/emails/exception-celery/content.txt

```diff
@@ -1,15 +0,0 @@
-{% autoescape off %}A Celery task raised a {{ exception_type }} exception.
```

Comment:
belongs to 729c4c493dbd38d9d3fa5b9cddc92761dc71416a, but I guess we're going to squash that anyways